README.md

```diff
@@ -0,0 +1,96 @@
+# Packetbeat
+
+Network data can tell you a detailed story of what's going on across your
+applications. Packetbeat is an open source project that is designed to provide
+real‑time analytics for web, database, and other network protocols. Whether
+you're in IT operations, security, or a business analyst, you can rapidly
+extend Packetbeat's functionality by adding new protocols, metrics, and
+analytics.
+
+## Usage
+
+Packetbeat can be added to any principal charm thanks to the wonders of being
+a subordinate charm. The following usage example will deploy the elk stack,
+so we can visualize our wire data once we've established the link between
+packetbeat and Logstash
+
+    juju deploy cs:~containers/bundle/elk-stack
+    juju deploy cs:trusty/consul
+    juju deploy cs:trusty/packetbeat
+    juju add-relation packetbeat:beats-host consul
+    juju add-relation packetbeat logstash
+
+
+### Deploying the minimal Beats formation
+
+If you do not need log buffering and alternate transforms on your data thats
+being shipped to ElasticSearch you can simply deploy the 'beats-base' bundle
```

Comment:
I think this actually touches the other beats readme's as well... *looks*  this was largely a copy/paste and chop operation